veil-tests/src/compile_tests.rs

```diff
@@ -183,4 +183,13 @@ fn test_redact_multiple_attributes() {
         hidden: bool,
         exposed: bool,
     }
+
+    #[derive(serde::Serialize, Redact)]
+    #[serde(rename_all = "camelCase")]
+    #[redact(all)]
+    struct MultipleAttributesAll {
+        #[serde(default)]
+        hidden: bool,
+        exposed: bool,
+    }
```

Comment:
Should these tests also verify that serde attributes still get applied? Or maybe doing this in a separate integration test
```suggestion
    let json = serde_json::json!({
        "exposed": true
    });
    let attributes: MultipleAttributesAll = serde_json::from_value(json).unwrap();
    assert_eq!(attributes.exposed, true);
    assert_eq!(attributes.hidden, false);
}
```
(this suggestion won't compile as it needs serde_json as well)